### ISSUE TITLE
OSX GUI Installer bug, "cannot install Anaconda3 in this location"

### ISSUE THREAD

**Issue by vestuto** [ContinuumIO/anaconda-issues#836]
* Encountered on OSX 10.9, 10.10, 10.11
* Encountered with both Anaconda3 and Anacodna2 installers
* There is a work around, but it makes the installer look janky
* When selecting the "Install only for me" option, the installer GUI reports "You cannot install Anaconda3 in this location", and the user is prevented from installing under their home path.
* Workaround: if you select any other install option on the same window, and then *reselect* the option "Install only for me" then the error message goes away, and the user is no longer prevented from installing in the users home path.

Attached images of GUI installer behavior.

<img width="623" alt="anaconda_install_gui_bug_pic01" src="https://cloud.githubusercontent.com/assets/6679073/16234327/c6a192f4-3796-11e6-945a-aab8187bd918.png">
<img width="624" alt="anaconda_install_gui_bug_pic02" src="https://cloud.githubusercontent.com/assets/6679073/16234328/c6ae889c-3796-11e6-8255-d3b3e6539088.png">
<img width="623" alt="anaconda_install_gui_bug_pic03" src="https://cloud.githubusercontent.com/assets/6679073/16234330/c6bf07b2-3796-11e6-837b-1a5e538b0e5b.png">

**Comment by kerrywatson1** [comment]
ON https://docs.continuum.io/anaconda/install#anaconda-for-os-x-graphical-install
Anaconda docs do cover this with, 'IMPORTANT: If you get the error message “You cannot install Anaconda in this location”, reselect “Install for me only”.' 

BUT fixing the bug would be a better solution. This bug has been around a long time.

**Comment by Kyle-Falconer** [comment]
Still experiencing this issue, more than a year later.

**Comment by msarahan** [comment]
This is a known issue with Mac pkg installers.  We have raised support issues with Apple and received no response.  Long-term, we are working on alternate installer technologies, but have no definite timeline at this point.  From https://stackoverflow.com/a/11487658/1170370,

> You want to install items into the current user’s home folder.

> Short answer: DO NOT TRY IT!

> Long answer: REALLY; DO NOT TRY IT! Read Installer Problems and Solutions. You know what I did even after reading this? I was stupid enough to try it. Telling myself I'm sure that they fixed the issues in 10.7 or 10.8.

> First of all I saw from time to time the above mentioned Destination Select Pane Bug. That should have stopped me, but I ignored it. If you don't want to spend the week after you released your software answering support e-mails that they have to click once the nice blue selection DO NOT use this.

> You are now thinking that your users are smart enough to figure the panel out, aren't you? Well here is another thing about home folder installation, THEY DON'T WORK!

> I tested it for two weeks on around 10 different machines with different OS versions and what not, and it never failed. So I shipped it. Within an hour of the release I heart back from users who just couldn't install it. The logs hinted to permission issues you are not gonna be able to fix.

> So let's repeat it one more time: We do not use the Installer for home folder installations!

**Comment by msarahan** [comment]
Note that .sh installers do not have this issue, and may be a better option for you until we can get an alternate technology deployed.

**Comment by Kyle-Falconer** [comment]
I totally agree that Anaconda should not be in the User directory. I don't know how it would work with switching Python contexts, but I think it should be installed in the `/Applicaitons` folder

**Comment by hmanz** [comment]
When installing Anaconda3, the "Install for me only" option shows that it will be installed in "Macintosh HD" folder instead of users/home-dir folder. Is this normal?
